examples_and_tutorials/notebooks/active_vision/candidate_selection.ipynb

```diff
@@ -0,0 +1,138 @@
+{
```

Comment:
what does this notebook do?

Reply:
deleted it and added tests as discussed.